lib/pos_blockchain.go

```diff
@@ -66,6 +137,7 @@ func (bc *Blockchain) processBlockPoS(desoBlock *MsgDeSoBlock, currentView uint6
 	//     set for the block, we can at least store it locally.
 	// 2. Validate QC
 	if err = bc.validateQC(desoBlock, validatorsByStake); err != nil {
+		// TODO: If this fails, do we want to store the block as Stored & ValidateFailed?
```

Comment:
Yes, if the block fails QC validation, then it fits the criteria of ValidateFailed. We want to store it as such because we'll know next time we see this block, or see a child of this block that we don't want to try to process it again.